infra/Makefile

```diff
@@ -77,9 +80,10 @@ docker_test_lint_gha:
 .PHONY: docker_generate_docs
```

Comment:
**Question:**
Is this `docker_generate_docs` command something maintainers (e.g., my team) should be running manually and periodically to keep the docs up-to-date?